### DISCUSSION TITLE
Option to list Success Calls from Tab

### DISCUSSION THREAD

**Discussion by aimtiaz11** [DannyDainton/newman-reporter-htmlextra#407] [Ideas]
Currently we have the ability to see the tabs: Summary, Total Request, Failed Requests, and Skipped Tests. 
I think its good idea to have "Passed Requests" as well. 

There are cases where, when reporting on testing outcome, its easier to say "We ran 100 tests, only these 2 have passed, rest failed".
This tab would give that useful information when most tests have failed.

**Comment by DannyDainton** [comment]
The report is just a template file that you can copy and change to display any information from the Newman object. If you wanted to add something, take it away, or change the name of anything it's there for you to do. 

You can make a copy of this file https://github.com/DannyDainton/newman-reporter-htmlextra/blob/main/lib/dashboard-template.hbs, make changes and then run it in the command like this:

```
newman run collection.json -r htmlextra --reporter-htmlextra-template myTemplate.hbs
```
